packages/shared/lib/core/wallet/utils/getOutputParameters.ts

```diff
@@ -103,7 +109,7 @@ async function buildOutputParametersForLayer2(transactionDetails: NewTransaction
 
     let estimatedData = await getEstimateData()
 
-    if (estimatedData.gasEstimatePayload.gasBurned) {
+    if (estimatedData?.gasEstimatePayload?.gasBurned) {
```

Comment:
If there is no `gasBurned`, the `outputParams` are not returned in the `buildOutputParametersForLayer2` function.
As in the previous comment, I think we should handle it and show an error notification, to avoid a crash in the future. In this case, we then try to get the amount of the `outputParams`, but since it is `undefined`, it crashes :trollface: 
![Captura de pantalla de 2023-11-16 15-20-09](https://github.com/iotaledger/firefly/assets/100352899/f14bc452-47a1-499c-9783-8b911ec98579)

Reply:
ok found it, it was a promise reject that was crashing everything, should be fixed now